test/commands/core/corehelpers.ts

```
@@ -0,0 +1,116 @@
+import { expect } from "chai"
+import { runCli } from "../../runCli"
+
+// Helper to strip ANSI color codes
+const stripAnsi = (str: string) => str.replace(/\u001b\[\d+m/g, '')
```

Biome (1.9.4)

[error] Unexpected control character in a regular expression.

Control characters are unusual and potentially incorrect inputs, so they are disallowed.

(lint/suspicious/noControlCharactersInRegex)